src/utils.c

```diff
@@ -672,6 +672,12 @@ time_t get_today(void)
 	return date2sec(day, 0, 0);
 }
 
+/* Returns the beginning of the selected day in the calendar. */
+time_t get_slctd_day(void)
+{
+	return date2sec(*ui_calendar_get_slctd_day(), 0, 0);
+}
+
```

Comment:
Not sure whether I like that. The idea was to have all functions directly accessing UI properties named with a `ui_` prefix. I agree that `ui_calendar_get_slctd_day_sec()` might not have been the best name but the new one is inconsistent with the remaining naming. I also believe it belongs to some of the UI units and not to the general utility functions...

Talking about naming: is there any reason the subject line of this commit starts with `Multiple days`? I don't see how this is related to the multiple days feature. You might have done it as a preparatory cleanup to that series but I think it might cause confusion instead of adding value for future readers of the commit history...

Reply:
I see your point about `ui_`. I treated the selected day like the current day. After all, the two variables

`static struct date today, slctd_day;`

live side by side in `ui-calendar.c`. Would you prefer `ui_get_slctd_day()`? in `ui_calendar.c`?

Reply:
Yes, `ui_get_slctd_day()` sounds good.

Reply:
I would rather not introduce name changes at this stage (experience has taught me that). Both variables should be looked at when this PR is closed.